packages/web-app-files/tests/unit/components/Collaborators/Collaborator.spec.js

```diff
@@ -183,11 +193,154 @@ describe('Collaborator component', () => {
         )
       })
     })
+    describe('reshare information', () => {
+      it.each([undefined, 0, null, false, '0', '', []])(
+        "does not show information if it's not a reshared share",
+        resharers => {
+          const wrapper = createWrapper({
+            shareType: 0,
+            collaborator: {
```

Comment:
What do you think about extracting via `collaborator: exampleCollaborator` which is defined above the tests?

Reply:
good point, I moved a default collaborator to be created in `createWrapper`, code is much shorter now